documentation/puppet-api/v3/plan_detail.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: default
+title: "Puppet Server: Puppet API: Plan detail"
+canonical: "/puppetserver/latest/puppet-api/v3/plan-detail.html"
+---
+
+### Uses `application/json` Content-Type
+
+The Content-Type in the response to an plan API query is
+`application/json`.
+
+## `GET /puppet/v3/plans/:module/:plan?environment=:environment`
+
+(Introduced in Puppet Server 5.1.0.)
```

Comment:
I think puppetserver got tagged this week for the platform release, so we can find the right 6.Y?

Reply:
Good catch, looks like 6.6.0, updated!